builq.go

```diff
@@ -13,10 +13,23 @@ type Builder struct {
 	pp    placeholderProvider
 }
 
-func NewPostgreSQL() *Builder { return &Builder{pp: &postgres{}} }
-func NewMySQL() *Builder      { return &Builder{pp: &mysql{}} }
+func NewIterBuilder(placeholder string) Builder {
```

Comment:
Should we return a pointer to `Builder` because the `Appendf` method uses pointer semantics?

Reply:
Not sure, it works well & aligns with `var b builq.Builder`.